samples/native_module_host/src/native_host_sample_lin.json

```diff
@@ -0,0 +1,41 @@
+{
+  "modules": [
+    {
+      "name": "out of process",
+      "loader": {
+        "name": "outprocess",
+        "entrypoint": {
+          "activation.type": "none",
+          "control.id": "outprocess_module_control"
+        }
+      },
+      "args": {
+        "outprocess.loader" : {
+          "name": "native",
+          "entrypoint": {
+             "module.path": "../../modules/logger/liblogger.so"
+          }
+        },
+        "module.arguments" : {
+          "filename": "log.txt"
+        }
+      }
+    },
+    {
+      "name": "hello_world",
+      "loader": {
+        "name": "native",
+        "entrypoint": {
+          "module.path": "../../modules/hello_world/libhello_world.so"
+        }
+      },
+      "args": null
+    }
+  ],
+  "links": [
+    {
+      "source": "hello_world",
+      "sink": "logger"
```

Comment:
I think this is wrong. the name of the module right now is "out of process". We shall rename to oop-logger... on both places.